tower-blocking-service/src/lib.rs

```diff
@@ -0,0 +1,47 @@
+use lazy_static::lazy_static;
+use std::sync::{Arc, Mutex};
+use tokio::runtime::Runtime;
+use tower_service::Service;
+use tower_service_util::BoxService;
+
+lazy_static! {
+    static ref RUNTIME: Arc<Mutex<Runtime>> = Arc::new(Mutex::new(
+        Runtime::new().expect("Runtime could not be created")
+    ));
+}
+
+trait BlockingService<Request> {
+    type Response;
+    type Error;
+
+    fn call(&mut self, req: Request) -> Result<Self::Response, Self::Error>;
+}
+
+struct BlockingSvc<T, U, E> {
+    inner: BoxService<T, U, E>,
```

Comment:
I think this needs to be Boxed because the result of a `ServiceBuilder` isn't nameable at the moment.